machines/Issuers/IssuersActions.ts

```diff
@@ -254,6 +239,11 @@ export const IssuersActions = (model: any) => {
       );
     },
 
+    updateVerificationErrorMessage: assign({
```

Comment:
In verify credential please return Technical error message directly as we are already logging the error which we are getting from the library so no need to return that error and log it again in the machines.

Reply:
Done